src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+
+function convertNumber(input) {
+  const convertedNumber = +input.slice(1).replace(',', '');
+
+  return convertedNumber;
+}
+
+function sortList(items) {
+  const itemsList = [...items.children];
+
+  const sortedItems = itemsList.sort(
+    (a, b) => convertNumber(b.dataset.salary) - convertNumber(a.dataset.salary),
```

Comment:
The sorting function assumes that each `li` element has a `data-salary` attribute. Ensure this attribute is present and correctly formatted as a string that can be converted to a number.